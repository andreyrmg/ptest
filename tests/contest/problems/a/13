100
14 AC
10 816
24 1A4
34 JK
31 QH
27 1A1
21 151
13 436
15 1C9
10 385
14 11
11 296
12 524
13 241
11 477
13 455
29 R1
29 DE
9 413
15 1A8
10 812
36 N9
5 11322
20 5H
21 7F
10 959
19 E7
10 75
2 100111010
20 5F
27 AB
12 204
16 CF
19 1E3
6 444
2 1011011101
29 IE
27 68
11 572
9 1258
5 12140
3 1002021
31 GI
7 511
16 5E
33 JU
3 210002
30 71
25 10B
22 5
3 222102
2 101111
15 37
15 1AC
24 1GD
4 33121
16 2F4
10 118
35 EG
12 23A
32 3R
27 4
22 L2
28 K3
32 H1
10 847
18 221
19 1E3
26 N7
35 1U
34 1W
28 F9
22 K3
23 1I6
29 1B
9 486
8 617
34 3C
29 5
16 241
35 4G
18 26B
10 640
26 3M
11 490
31 I0
12 36B
12 230
26 L1
6 1442
26 DE
15 3CD
35 5T
16 A5
5 11240
4 133
10 481
36 FP
7 2130
27 2Q
