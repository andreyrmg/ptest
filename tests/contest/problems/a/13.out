27 4
22 5
29 5
14 11
4 133
29 1B
2 101111
15 37
35 1U
34 1W
10 75
27 2Q
16 5E
26 3M
34 3C
20 5F
20 5H
10 118
32 3R
14 AC
35 4G
21 7F
16 A5
27 68
6 444
35 5T
16 CF
30 71
7 511
19 E7
27 AB
12 204
2 100111010
12 230
12 23A
9 413
11 296
26 DE
15 1A8
10 385
6 1442
15 1AC
13 241
29 DE
8 617
9 486
15 1C9
28 F9
22 K3
22 L2
10 481
35 EG
31 GI
12 36B
29 IE
32 H1
21 151
26 L1
31 I0
28 K3
36 FP
11 477
3 210002
16 241
11 490
26 N7
19 1E3
19 1E3
25 10B
10 640
33 JU
34 JK
11 572
18 221
3 222102
13 436
2 1011011101
13 455
12 524
7 2130
16 2F4
18 26B
29 R1
3 1002021
10 812
10 816
5 11240
24 1A4
31 QH
5 11322
36 N9
10 847
15 3CD
5 12140
9 1258
23 1I6
10 959
24 1GD
4 33121
27 1A1
